backend/cron.js

```diff
@@ -0,0 +1,92 @@
+const PullRequest = require('./services/PullRequest')
+const redisClient = require('./redis')
+
+// command line: nodejs cron.js startDate endDate
+const args = process.argv.slice(2) || []
+
+let startDate = args[0] || '2019-10-01 00:00:00'
+const endDate = args[1] || '2019-10-31 23:59:59'
```

Comment:
Oh, we have default values. Nice. :smiley: 

I was thinking of passing the year and just fetch october. But we can do it this way. :)